source/core/TDCodecVia.cpp

```diff
@@ -506,23 +506,31 @@ void TDViaParser::ParseAggregateElementList(std::vector<TypeRef> *elementTypesVe
     SubString  token;
     SubString  fieldName;
     UsageTypeEnum  usageType;
+    bool isDataItem;
 
     _string.ReadToken(&token);
     if (!token.CompareCStr("("))
         return LOG_EVENT(kHardDataError, "'(' missing");
 
     _string.ReadToken(&token);
     while (!token.CompareCStr(")")) {
+        isDataItem = false;
         if (token.CompareCStr(tsElementToken)) {
             usageType = kUsageTypeSimple;
+        } else if (token.CompareCStr(tsDataitemElementToken)) {
+            usageType = kUsageTypeSimple;
+            isDataItem = true;
         } else if (token.CompareCStr(tsConstElementToken)) {
             usageType = kUsageTypeConst;
         } else if (token.CompareCStr(tsInputParamToken)) {
             usageType = kUsageTypeInput;
+            isDataItem = true;
```

Comment:
@rajsite  Did you discover a reason inputs need to be dataItems?
The code which inserts the call to SetValueNeedsUpdate does it on the destination of a copy, which I think can never be an input.

Reply:
This is unchanged from the [previous PR](https://github.com/ni/VireoSDK/pull/590/files#diff-e8e3eac825feec1ddca7e0d263928ecdR527) made in [this commit](https://github.com/ni/VireoSDK/pull/590/commits/fef7e10c025ab8788aae3ff92f537bac7ceba9f9).

Reply:
Ah, ok, there's a conversation about why this change was made in that [commit](https://github.com/ni/VireoSDK/pull/590/commits/fef7e10c025ab8788aae3ff92f537bac7ceba9f9) that I had completely forgotten about.   It's all good, then.